.travis.yml

```diff
@@ -40,11 +41,10 @@ matrix:
       env: JRUBY_OPTS='--dev --1.8'
     - rvm: 2.7.1
       env: DIFF_LCS_VERSION="~> 1.3.0"
-    - rvm: 2.7.1
-      env: DIFF_LCS_VERSION="1.4.3"
```

Comment:
What we do fo this change?

Reply:
It needs to be here, but I don't mind if its also in the other repos.